api/Dockerfile

```diff
@@ -95,69 +86,52 @@ RUN set -eux; \
 	composer install --prefer-dist --no-dev --no-scripts --no-progress; \
 	composer clear-cache
 
-COPY .env ./
-RUN composer dump-env prod
-
 # copy only specifically what we need
+COPY .env ./
 COPY bin bin/
 COPY config config/
-COPY data data/
 COPY migrations migrations/
 COPY public public/
 COPY src src/
+COPY templates templates/
 # only for demo
 COPY fixtures fixtures/
 
 RUN set -eux; \
 	mkdir -p var/cache var/log; \
 	composer dump-autoload --classmap-authoritative --no-dev; \
+	composer dump-env prod; \
 	composer run-script --no-dev post-install-cmd; \
 	chmod +x bin/console; sync
 VOLUME /srv/api/var
 
-COPY docker/php/docker-healthcheck.sh /usr/local/bin/docker-healthcheck
-RUN chmod +x /usr/local/bin/docker-healthcheck
-
-HEALTHCHECK --interval=10s --timeout=3s --retries=3 CMD ["docker-healthcheck"]
+COPY docker/php/php-fpm-healthcheck /usr/local/bin/php-fpm-healthcheck
+RUN chmod +x /usr/local/bin/php-fpm-healthcheck
+HEALTHCHECK --interval=10s --timeout=3s --retries=3 CMD ["php-fpm-healthcheck", "-v"]
 
 COPY docker/php/docker-entrypoint.sh /usr/local/bin/docker-entrypoint
 RUN chmod +x /usr/local/bin/docker-entrypoint
 
+ENV SYMFONY_PHPUNIT_VERSION=9
+
 ENTRYPOINT ["docker-entrypoint"]
 CMD ["php-fpm"]
 
-
-# "nginx" stage
+# "caddy" stage
 # depends on the "php" stage above
-# The OpenResty distribution of NGINX is only needed for Kubernetes compatiblity (dynamic upstream resolution)
-FROM openresty/openresty:${OPENRESTY_VERSION}-alpine AS api_platform_nginx
+FROM caddy:${CADDY_VERSION}-builder-alpine AS api_platform_caddy_builder
 
-RUN echo -e "env UPSTREAM;\n$(cat /usr/local/openresty/nginx/conf/nginx.conf)" > /usr/local/openresty/nginx/conf/nginx.conf
-COPY docker/nginx/conf.d/default.conf /etc/nginx/conf.d/default.conf
+RUN xcaddy build \
+    --with github.com/dunglas/mercure \
+    --with github.com/dunglas/mercure/caddy
+    #--with github.com/dunglas/vulcain \
+    #--with github.com/dunglas/vulcain/caddy
 
-WORKDIR /srv/api/public
+FROM caddy:${CADDY_VERSION} AS api_platform_caddy
 
-COPY --from=api_platform_php /srv/api/public ./
-
-
-# "varnish" stage
-# does not depend on any of the above stages, but placed here to keep everything in one Dockerfile
-FROM varnish:${VARNISH_VERSION} AS api_platform_varnish
-
-ENV BACKEND=api
-ENV UPSTREAM=php
-
-COPY docker/varnish/conf/default.tmpl /usr/local/etc/varnish/default.tmpl
-
-RUN set -eux; \
-	apt-get update; \
-	apt-get install -y gettext-base; \
-	apt-get clean; \
-	rm -rf /var/lib/apt/lists/*
-
-COPY docker/varnish/docker-entrypoint.sh /usr/local/bin/docker-entrypoint
-RUN chmod +x /usr/local/bin/docker-entrypoint
-
-ENTRYPOINT ["docker-entrypoint"]
+WORKDIR /srv/api
 
-CMD ["varnishd", "-F", "-f", "/etc/varnish/default.vcl", "-p", "http_resp_hdr_len=65536", "-p", "http_resp_size=98304"]
+COPY --from=dunglas/mercure:v0.11 /srv/public /srv/mercure-assets/
```

Comment:
```suggestion
COPY --from=dunglas/mercure:v0.11 /srv/public /srv/mercure/
```

?

Reply:
No, according to https://github.com/dunglas/symfony-docker/blob/main/Dockerfile#L125 and https://github.com/api-platform/api-platform/blob/main/api/Dockerfile#L133, it is the correct path